cmd/snap/cmd_abort.go

```diff
@@ -25,11 +25,7 @@ import (
 	"github.com/snapcore/snapd/i18n"
 )
 
-type cmdAbort struct {
-	Positional struct {
-		ID changeID
-	} `positional-args:"yes" required:"yes"`
-}
+type cmdAbort struct{ lastMixin }
```

Comment:
To keep things interesting, I'll mention that last is also a noun.

the PR description is a bit cryptic, but the commits are nicely separate I think?